zkevm-circuits/src/state_circuit/state.rs

```diff
@@ -1486,8 +1472,7 @@ mod tests {
     #[test]
     fn first_write() {
         let stack_op_0 = StackOp::new(
```

Comment:
Why not remove `stack_op_0` entirely? Or it's for dummy usage?

Reply:
Thanks! Yes, not sure if it makes much sense to leave it there, but the idea was to have one test which has no memory ops, but some stack and storage ops. There is also, for example, a test "storage" which has no memory and stack ops. These tests (first_write and storage) actually test other things, but I thought it's good to cover cases where there are empty ops of some type(s). Do you think it's better to remove it?

Reply:
Ah okay, make sense to me now. But it reminds me that we didn't have the failure case that the memory is read when memory address changes (so as a first writes). It would be good to include that in this test, right? (we could add it further PR for sure)

Reply:
Ah okay, make sense to me now. But it reminds me that we didn't have the failure case that the memory is read when memory address changes (so as a first writes). It would be good to include that in this test, right? (we could add it further PR for sure)

Reply:
Yes, I was thinking about that some time ago. There is no such test because "init" memory rows are inserted automatically and I would need to pass the information to the circuit about not automatically inserting "init" rows to enable such test. Do you think we should do this?

Reply:
Ah I see, maybe just keep it as is now.